### ISSUE TITLE
Chapter 1.2 images and text 

### ISSUE THREAD

**Issue by TomFranklin** [dfe-analytical-services/r-training-course#13]
The image for the RStudio layout already contains a project loaded and data in the environment and describes the layout as having a script in the top left. 

I'd suggest this to be changed to RStudio from the beginning, then creating a project then a new script and loading data in chapter 2. I'd also suggest a small section about zooming / global options, as DfE laptops are wild for making RStudio zoom tiny and massive.

**Comment by matt-dray** [comment]
Yes, let's assume they're starting from absolute scratch. Any replacement screenshots should use default RStudio theming, etc.

Would be good to get more explanation on Project benefits here. Ditto setting up the subfolders in the Project directory. Do we introduce that here? Do we intrioduce `dfeR` as a way of doing this? (Might be too much this early on -- could go in a tip box or something.) Maybe these are separate issues.

**Comment by TomFranklin** [comment]
I do think incorporating the `dfeR` package right from the beginning makes sense, helps with consistency. Would need to think about how it's functions link with existing material. But considering it is a DfE R training guide I think it'd be a good addition!
